ReactWindows/ReactNative.Shared/Bridge/ReflectionReactDelegateFactory.cs

```diff
@@ -165,6 +168,13 @@ private Extractor CreateExtractor(Type type, string moduleName, string methodNam
                 idx = result.NextIndex;
             }
 
+            bool hasDelegate = (genericDelegate != null) && ReferenceEquals(moduleInstance, genericDelegate.DelegateTarget);
+            if (hasDelegate)
+            {
+                genericDelegate.Invoke(args);
+                return;
+            }
```

Comment:
I think I mentioned this in the last PR, but let's create a new `IReactDelegateFactory` - `GenericReactDelegateFactory` that has this logic.